src/tools/rbd_mirror/PoolReplayer.cc

```diff
@@ -550,6 +550,16 @@ int PoolReplayer<I>::init_rados(const std::string &cluster_name,
     }
   }
 
+  auto seconds = cct->_conf.get_val<std::chrono::seconds>(
+      "rbd_mirror_remote_osd_op_timeout").count();
+  r = cct->_conf.set_val("rados_osd_op_timeout", std::to_string(seconds));
```

Comment:
This would apply the timeout intended only for the remote client also to the local client.

I think this needs to be conditioned on `strip_cluster_overrides` (it's probably a good idea to rename it to something like `is_remote` while at it) .